shell/platform/embedder/embedder.h

```diff
@@ -1673,6 +1675,16 @@ typedef struct {
   /// This represents the refresh period in frames per second. This value may be
   /// zero if the device is not running or unavailable or unknown.
   double refresh_rate;
+
+  /// The width of the display, in physical pixels.
+  double width;
+
+  /// The height of the display, in physical pixels.
+  double height;
```

Comment:
Just to double-check, the width/height types (`double`) are intentionally different from the window metric event's width/height types (`size_t`)?

Reply:
No. I'll change to size_t